src/components/Search.jsx

```diff
@@ -11,20 +11,29 @@ export class Search extends Component {
     this.setState({ value: e.target.value })
   }
 
+  rmSpecChars(word) {
+    return word.split('').filter((ltr) => {
+      return !["'", " ", "-", ",", ":", "#", ".", "!", "?"].includes(ltr)
+    }).join('')
```

Comment:
nice job splitting this method up, i think it was getting too long before. good thinking so we can follow SRP.